### PR TITLE
updated testscript for add VN and LR

### PR DESCRIPTION
**Pull request by sureshkvl**

### CODE REVIEW

**Comment by nlewo** [comment]
@sureshkvl you would also have to check a route-target has been created in the logical router routing-instance.

File: test/load-database.nix [hunk 1]
```diff
@@ -57,6 +57,10 @@ let
   testScript = ''
     $machine->waitForOpenPort(8082);
     $machine->waitUntilSucceeds("${contrailPkgs.tools.contrailApiCliWithExtra}/bin/contrail-api-cli ls -l virtual-network | grep -q vn1");
+    $machine->waitUntilSucceeds("${contrailPkgs.tools.contrailApiCliWithExtra}/bin/contrail-api-cli --ns contrail_api_cli.provision add-vn --project-fqname default-domain:default-project testvn2");
```
Comment (eonpatapon) [inline test/load-database.nix]: should be `->succeed`

**Comment by eonpatapon** [comment]
I think it's simpler to check for VN RT because you know VN RI name in advance
You don't need to create LR laso

**Comment by sureshkvl** [comment]
Please ignore this pull request. Ihave created another pull request(rebased  with latest code, and addressed the review comments)

**Pull request closed without merging**